src/crud.js

```diff
@@ -71,13 +72,28 @@ export default (server, model, { prefix, defaultConfig: config, models: permissi
     return params;
   }, {});
 
-  const validAssociations = modelAssociations.length
-    ? joi.string().valid(...modelAssociations)
-    : joi.valid(null);
+  const modelsHasAssociations = modelAssociations && modelAssociations.length;
+  const validAssociationsString = modelsHasAssociations
+      ? joi.string().valid(...modelAssociations)
+      : joi.valid(null);
+  const validAssociationsObject = modelsHasAssociations
+      ? joi.object().keys({
+        model: joi.string().valid(...modelAssociations),
+        where: joi.object().keys({
+          ...attributeValidation,
+          ...sequelizeOperators,
+        }),
+        as: joi.string().valid(...associatedModelAliases),
+        include: joi.any(), // @Todo: should validate the same as associationValidation var below
```

Comment:
@joeybaker and @mdibaiee, i need help to apply include child joi validation to be the same as its parent include validation (just like nested validations with the same schema)